python/ray/experimental/state.py

```diff
@@ -569,3 +638,27 @@ def workers(self):
                 "stdout_file": worker_info[b"stdout_file"].decode("ascii")
             }
         return workers_data
+
+    def job_length(self):
```

Comment:
When I call this function without running any tasks, it returns `(9223372036854775807, 0, 0)`. That probably isn't what we want.